tools/github_actions_dependencies.sh

```diff
@@ -14,8 +14,9 @@ else
 	echo "Date utils"
 	# https://pip.pypa.io/en/latest/user_guide/#possible-ways-to-reduce-backtracking-occurring
 	pip install $STD_ARGS --pre --only-binary ":all:" python-dateutil pytz joblib threadpoolctl six
-	echo "PyQt6"
-	pip install $STD_ARGS --pre --only-binary ":all:" --no-deps --extra-index-url https://www.riverbankcomputing.com/pypi/simple PyQt6 PyQt6-sip PyQt6-Qt6
+	# XXX: the wheels for PyQt6-sip 13.4 are not available yet from https://www.riverbankcomputing.com/pypi/simple
+	# echo "PyQt6"
+	# pip install $STD_ARGS --pre --only-binary ":all:" --no-deps --extra-index-url https://www.riverbankcomputing.com/pypi/simple PyQt6 PyQt6-sip PyQt6-Qt6
```

Comment:
Actually for now we probably *do* still want to install some variant of Qt, so maybe just `pip install --only-binary ":all:" PyQt6 PyQt6-sip PyQt6-Qt6` would be good here

Reply:
I'll update right away